crates/forge-runner/src/gas.rs

```diff
@@ -163,11 +163,31 @@ pub fn check_available_gas(
             arguments,
             gas_info,
             ..
-        } if available_gas.is_some_and(|available_gas| gas_info > available_gas as u128) => {
+        } if available_gas.is_some_and(|available_gas| match available_gas {
+            RawAvailableGasConfig::MaxGas(gas) => {
+                print_as_warning(&anyhow!(
+                    "Setting available_gas with unnamed argument is deprecated. \
+                Consider setting resource bounds (l1_gas, l1_data_gas and l2_gas) explicitly."
```

Comment:
Please create an issue to remove this following deprecation (can be tentatively scheduled for iteration 9) and link it here.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/3109